plugin/Scripts/FrameVisualizer.cs

```diff
@@ -10,7 +10,7 @@ public class FrameVisualizer : MonoBehaviour
         public Transform cameraAsParent;
         public Material eyeFrameMaterial;
         [Header("Settings")]
-        [Tooltip("Used for adjusting the Eye Frames, restart needed!")] 
+        [Tooltip("Used for adjusting the Eye Frames, restart needed!")]
```

Comment:
Please specify what needs restarting.
```suggestion
        [Tooltip("Used for adjusting the Eye Frames. Changes require restarting X!")]
```

Reply:
I edited your comment for easy "apply suggestion" ;)